### PR TITLE
[RF] Get rid of unused variables

### PR DESCRIPTION
**Pull request by kahinimehta**
- Remove variables that are not being used

### CODE REVIEW

File: xcp_d/workflow/restingstate.py [hunk 1]
```diff
@@ -167,7 +167,6 @@ def init_compute_alff_wf(mem_gb,
 
 def init_surface_reho_wf(mem_gb,
                          omp_nthreads,
-                         smoothing,
```
Comment (tsalo) [inline xcp_d/workflow/restingstate.py:170]: Can you also remove smoothing from the parameter list in the docstring?

File: xcp_d/workflow/postprocessing.py [hunk 2]
```diff
@@ -24,18 +24,12 @@
...
         upper_bpf,
         bpf_order,
         smoothing,
         bold_file,
         params,
-        motion_filter_type,
-        motion_filter_order,
-        band_stop_max,
-        band_stop_min,
-        initial_volumes_to_drop,
```
Comment (tsalo) [inline xcp_d/workflow/postprocessing.py:38]: Some of these need to be removed from the example call and the parameters list in the docstring.

File: xcp_d/interfaces/layout_builder.py [hunk 3]
```diff
@@ -256,11 +255,10 @@ def __init__(self, tasks=[], img_path='./figures', **kwargs):
 
         self.run(tasks)
 
-    def write_t1_reg_rows(self, task_name, task_num):
+    def write_t1_reg_rows(self, task_name):
         """Write T1w rows."""
         # Write the header for the next few rows.
-        self.section += constants.TASK_LABEL_ROW.format(task_name=task_name,
-                                                        task_num=task_num)
+        self.section += constants.TASK_LABEL_ROW.format(task_name=task_name)
```
Comment (tsalo) [inline xcp_d/interfaces/layout_builder.py:261]: Why remove `task_num` from this method?

**Review by tsalo — CHANGES_REQUESTED** [review]
A few minor things. I think in some cases you need to remove the arguments from the docstrings as well.

File: xcp_d/interfaces/layout_builder.py (hunk 3 above)
Comment (kahinimehta) [reply xcp_d/interfaces/layout_builder.py:261]: I don't think it was being used

**Review by tsalo — APPROVED** [review]
Happy to merge this once CI passes.

**Pull request merged**